x-pack/plugins/index_lifecycle_management/public/application/store/defaults/hot_phase.js

```diff
@@ -24,12 +25,14 @@ export const defaultHotPhase = {
   [PHASE_INDEX_PRIORITY]: 100,
   [PHASE_ROLLOVER_MAX_DOCUMENTS]: '',
 };
-export const defaultEmptyHotPhase = {
-  ...defaultHotPhase,
+
+export const defaultHotPhaseWhenEditing = {
```

Comment:
...but I feel like in this case, the differences are _more_ apparent, probably because they're more different than they are similar. Maybe I should just preserve this change but revert the changes where the variables have few differences. Thoughts?